src/serverdlg.cpp

```diff
@@ -37,6 +37,9 @@ CServerDlg::CServerDlg ( CServer* pNServP, CServerSettings* pNSetP, const bool b
 
     setupUi ( this );
 
+    // always start on the main tab
+    tabWidget->setCurrentIndex ( 0 );
+
     // set window title
```

Comment:
Does the Qt 6.5 bug also occur if this isn't set?

Reply:
That's `src/serverdlg.cpp`, not related to the client settings "My Profile" weirdness.

Reply:
Uh. Yes. It's a different file - so that's true.